tests/kernel/mem_protect/userspace/testcase.yaml

```diff
@@ -14,6 +14,11 @@ tests:
       - CONFIG_COMMON_LIBC_MALLOC_ARENA_SIZE=0
     platform_exclude:
       - ucans32k1sic
+      # Commit cb19bd4fc2541931beb0e1f157127f7a50a082d4
```

Comment:
that commit was wrong, you can't disable USERSPACE like this on the board level, it will still be enabled 

```
warning: USERSPACE (defined at arch/Kconfig:272) was assigned the value 'n' but got the value 'y'.
See http://docs.zephyrproject.org/latest/kconfig.html#CONFIG_USERSPACE and/or look up USERSPACE in
the menuconfig/guiconfig interface. The Application Development Primer, Setting Configuration
Values, and Kconfig - Tips and Best Practices sections of the manual might be helpful too.

```

Reply:
submitted this instead https://github.com/zephyrproject-rtos/zephyr/pull/70129